src/mlbstandings/updater.py

```
@@ -268,28 +315,29 @@ def _get_newest_league_day(self, column: List[SheetValue], league: str, first_da
+        url = f"https://docs.google.com/spreadsheets/d/{self.get_spreadsheet_id_for_year(year)}/edit"
         name = self._spreadsheet_name(year)
-        html_content = '''<html>
+        html_content = """<html>
   <head></head>
   <body>
     <p>See <a href="%s">%s</a>.</p>
   </body>
-</html>''' % (url, name)
-        msg.set_content(f'See {url}')
-        msg.add_alternative(html_content, subtype='html')
+</html>""" % (url, name)
```

Ruff (0.12.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)